### ISSUE TITLE
Lack of documentation in examples

### ISSUE THREAD

**Issue by Lacerda** [facebook/fixed-data-table#265]
The examples are great, but there is a lack of documentation and source code.
It would be really cool to know, for example, the json structure used for the example "With json data".
Or how to implement resizable columns with example source code.

**Comment by donnrri** [comment]
Yes I agree. Just some simple code to start us off. Please

**Comment by Robinson7D** [comment]
I agree that there should be code connected to the examples (preferably very concise code).

Though for those willing to dig a little, all of these examples seem to be available in the /site/examples folder of this project. 

For example, the JSON / Object structure is available here:
https://github.com/facebook/fixed-data-table/blob/master/site/examples/FakeObjectDataListStore.js

**Comment by donnrri** [comment]
Yes just found them , thanks for the examples. A link from the website could have avoided the confusion tho

**Comment by csman** [comment]
The link above is dead. It would be trivial to include the format of JSON used in the example, why not just make people's lives easier and show the format, in comments or something?

**Comment by Lacerda** [comment]
I believe you can find the same example here:
https://github.com/facebook/fixed-data-table/blob/master/examples/helpers/FakeObjectDataListStore.js
